x-pack/filebeat/input/netflow/convert.go

```diff
@@ -259,6 +262,17 @@ func flowToBeatEvent(flow record.Record) (event beat.Event) {
 		revPkts, hasRevPkts = getKeyUint64(flow.Fields, "reversePacketTotalCount")
 	}
 
+	if biflowDir, isBiflow := getKeyUint64(flow.Fields, "biflowDirection"); isBiflow && len(ecsSource) > 0 && len(ecsDest) > 0 {
+		client, server := ecsSource, ecsDest
+		// swap client and server if biflowDirection is reverse
+		if biflowDir == 2 {
+			client, server = server, client
+		}
+		event.Fields["client"] = client
+		event.Fields["source"] = server
```

Comment:
Is it possible you meant to assign `event.Fields["server"] = server` instead?

The `client = client` assignment looks like I would expect, however.

Reply:
Another point about this. Is it possible that the golden files should be re-generated?

I understand the goal of this code to be to populate (duplicate) client & server field sets with the source & destination. This is what I would expect.

However I don't see the client/server pairs show up in the golden files.

Reply:
good catch @webmat , that's a big mistake

Reply:
@adriansr I pushed a change to only swap the source and destination when biflowDirection=reverseIntiator and refrain from adding client/server. Does this make sense?

Reply:
The reason the sample data didn't change is because we don't have a good netflow sample from a biflow exporter. The one capture we have with `biflowDirection` appears to be from a Cisco Application Visibility and Control (AVC) service and has L7 details but not L3.

Reply:
Gotcha.

So none of the sample data is populating client and server?